src/main/java/seedu/address/logic/wanderlustlogic/wanderlustparser/WanderlustGoToCommandParser.java

```diff
@@ -7,14 +7,14 @@
 import seedu.address.logic.wanderlustlogic.wanderlustparser.exceptions.ParseException;
 
 /**
- * Parses input arguments and creates a new DeleteCommand object
+ * Parses input arguments and creates a new GoToCommand object
  */
 public class WanderlustGoToCommandParser implements WanderlustParserInterface<GoToCommand> {
 
     /**
      * Parses the given {@code String} of arguments in the context of the GoToCommand
      * and returns a GoToCommand object for execution.
-     * @throws seedu.address.logic.parser.exceptions.ParseException if the user input does not conform
+     * @throws ParseException if the user input does not conform
```

Comment:
Can use the {@code ParseException} syntax instead of just the string?

Reply:
ok changed.